bats_test/downloads/file-downloads/publlic_files.bats

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bats
+
+# Support lib
+load '/home/node/packages/node_modules/bats-support/load.bash'
+# File methods. DEPENDS on SUPPORT lib.
+load '/home/node/packages/node_modules/bats-file/load.bash'
+# Assertions.  DEPENDS on SUPPORT lib.
+load '/home/node/packages/node_modules/bats-assert/load.bash'
+
+DIR="$(cd "$(dirname "$BATS_TEST_FILENAME")" >/dev/null 2>&1 && pwd)"
```

Comment:
This line is:

 - `cd` into the parent of `${BATS_TEST_FILENAME}`
   - the `stdout` of `cd` is redirected to `/dev/null`
   - the `stderr` is redirected to `stdout`
     - if there was an error doing `cd`, then returns `pwd` to put in `DIR`
     - if `cd` had success, then that's the value to put in `DIR`
-------------------
So, is there a case where `${BATS_TEST_FILENAME}` has a wrong value?

Reply:
Can't we have a hardcoded `${BATS_TEST_BASENAME}`?

Reply:
This way the test ll always work as it fetches the absolute path dynamically